packages/babel-traverse/src/visitors.js

```diff
@@ -23,7 +23,7 @@ export function explode(visitor) {
   visitor._exploded = true;
 
   // normalise pipes
-  for (const nodeType in visitor) {
+  for (const nodeType of Object.keys(visitor)) {
```

Comment:
@loganfsmyth Since this is a pulic api, should I handle the case where `visitor` is null/undefined (that `for..in` handled by default)?

Reply:
Since we're doing `visitor._exploded` above we shouldn't need to worry about that case here.